CONTRIBUTING.md

```diff
@@ -29,28 +32,52 @@ paragraphs, each separated by a blank line. It should explain the problem you so
 
 For example:
 
+---
 ```
 :sparkles: feat(etl): add new data source for customer segmentation
 ```
+---
 
 If you have multiple commits in one merge request, you should squash them into a single commit with a suitable commit
 message when merging with the main branch. The commit message should summarize what changes you made in the merge
 request and why. It should also reference the issue that you resolved and include the ticket number from Jira. For
 example:
 
+---
 ```
 :recycle: refactor(ansible): move dev machines group vars to the role vars #123
 ```
+---
 
 This indicates that you resolved ticket number 123, which was about refactoring the variables of development machines.
 
 
 ## Branching Name Convention :twisted_rightwards_arrows:
 
 Before creating a feature or resolving a bug, it is required to create an issue on GitHub. This issue will be assigned a number, which should be included in the branch name, separated by hyphens (-). Here is an example of a valid Git branch name:
+---
 ```bash
 2-feature-add-contribution-documentation
 ```
+---
+
+## Git Rebase Policy :rotating_light:
+
+We follow a Git rebase policy to keep our Git history clean and easy to understand. When you are working on a feature or bug fix, you should create a feature branch based on the latest main branch. Then, you should make your changes in this branch and commit them as usual. Once you are done with your changes and ready to merge your branch into the main branch, you should rebase your branch on the latest main branch to make sure it is up to date and has no conflicts.
+
+
+---
+```go
+git fetch origin
+git rebase origin/main
+```
```

Comment:
Would you please explain this example more? I mean what is the name of source branch (what is the name of your current branch) and what do you want to do) and its destination?